src/components/CHeader.vue

```diff
@@ -15,6 +47,37 @@
 <script>
 export default {
   name: 'CHeader',
-  components: {}
+  components: {},
+  methods: {
+    onUpdateMessages() {
+      this.$emit('onUpdateMessages');
+    },
+    onUpdateRefreshTime() {
+      this.$emit('onUpdateRefreshTime', this.refreshTimeSelected);
+    }
+  },
+  data() {
+    return {
+      refreshTimeSelected: 10,
```

Comment:
I would prefer refresh time to be an input of CHeader to avoid setting the default value of 10 at 2 different places.